runtime/sema/type.go

```diff
@@ -3543,6 +3556,13 @@ func (t *CompositeType) getBaseCompositeKind() common.CompositeKind {
 	return common.CompositeKindUnknown
 }
 
+func isAttachmentType(t Type) bool {
```

Comment:
Maybe double check this function is used everywhere necessary, not just the first case